tx-parser/src/lib.rs

```diff
@@ -138,7 +138,7 @@ impl<I> Tx<I>
     }
 
     fn add() -> TxParser<Entity, I> {
-        fn_parser(Tx::<I>::add_, "[:db/add e a v tx?]")
+        fn_parser(Tx::<I>::add_, "[:db/add e a v]")
```

Comment:
Ah -- this is just the string description; the actual parser declaration is at https://github.com/mozilla/mentat/pull/265/files/7105861f047775c19fabf11148b8f0bc7dd95963#diff-5a81d11a5ad2928cc7cf2547288345eaL118.  Remove the optional, that'll drop the element from the tuple.